mlrun/projects/project.py

```diff
@@ -132,7 +132,7 @@ def new_project(
     :param secrets:      key:secret dict or SecretsStore used to download sources
     :param description:  text describing the project
     :param subpath:      project subpath (relative to the context dir)
-    :param skip_save:    do not save the created project in the DB
+    :param save:         whether to save the created project in the DB
```

Comment:
This is a non backwards compatible change - it's ok since this argument introduced in 1.1.0-rc5 (i.e. was not released in an official release yet) - but you must mention in the PR description that you're doing such change and why it's ok